dist/src/main/java/io/camunda/webapps/controllers/IndexController.java

```diff
@@ -5,22 +5,40 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.operate.webapp;
+package io.camunda.webapps.controllers;
+
+import static io.camunda.application.Profile.*;
```

Comment:
I think no rule enforces it, but can you ensure that it lists the used one, and not only `*`

Reply:
`import static io.camunda.application.Profile.*` should be deleted anyway